src/adapters/SensorApplicationInterface.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * CS 410 - Assignment 2 
+ * @author Daniyal Adzhiyev
+ * @version 1.0 05/28/2019
+ * 
+ * This Application uses the Adapter Software Pattern to create adapter's for multiple predefined sensor classes.
+ * Where the adapter classes implement the Sensor Application Interface, to 'adapt' the original sensor classes
+ * so that all of the adapter classes have the same methods needed to display the information in the Sensor 
+ * Application.
+ *  
+ */
+
+package adapters;
+
+import java.awt.Color;
+
+/*
+ * This is the interface for the Sensor Application to communicate with the Sensor Adapter's
+ */
+public interface SensorApplicationInterface {
+
+	public String getSensorName(); // gets name of the sensor
+
+	public Color getColorCode(); // gets color code based on the severity of the sensor reading
```

Comment:
I would recommend moving most of these methods to SensorApplication or another class entirely. The adapters should only have methods to adapt methods from the sensors classes, and methods such as getLenth() and getColorCode() are very specific to the SensorApplication class. It would also lower duplication since you would not have to write the methods implementation in each adapter.